packages/amesos2/src/basker/basker.hpp

```diff
@@ -39,18 +39,18 @@
 #include "matrix.h"
 #include "mex.h"
 
-#define ASSERT(a) mxAssert(a,"")
-#define REALLOC(ptr,size) mxRealloc(ptr, size)
-#define CALLOC(num, size) mxCalloc(num, size)
+#define BASKERASSERT(a) mxAssert(a,"")
+#define BASKERREALLOC(ptr,size) mxRealloc(ptr, size)
+#define BASKERCALLOC(num, size) mxCalloc(num, size)
 #define BASKERFREE(ptr)
 
 #else
 #include <assert.h>
 #include <stdlib.h>
```

Comment:
The `<xxx.h>` C standard library headers were deprecated in C++ as of C++98; please use the `<cxxx>` headers instead.